documentation.html

```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
             </div>
           </section>
         </section>
+        <!-- Toggle button class starts here -->
+        <section class="instruction-block" id="toggle">
+          <h2 id="base-icon-btn">Toggle Button</h2>
+          <div class="text">
+            The <code>toggle-btn</code> class along with <code>toggle-off-btn</code> / <code>toggle-on-btn</code> state class enables you to have a toggle button.Apart from its basic use, it can be used for checkboxes and radiobuttons.
+          </div>
+          <br />
+          <section class="instruction-block" id="toggleCheckbox">
+            <h4 class="display-inlineBlock">Toggle Button as Checkboxes</h4>
+            <div class="text">
+              Adding the <code>toggle-btn toggle-off-btn</code> class to your checkbox inputs converts them to toggle buttons.For example:
```

Comment:
`toggle-off-btn` is not required for checkboxes and radio buttons. Only for normal buttons